packages/server/src/db/index.js

```diff
@@ -1046,7 +1044,22 @@ async function getGrant({ grantId }) {
 
 async function getSingleGrantDetails({ grantId, tenantId }) {
     const results = await knex.table(TABLES.grants)
-        .select('*')
+        .select([
```

Comment:
Making this change because @TylerHendrickson suggested that we not return `raw_body` or `raw_body_json` in API responses but also because I agree with the conventional wisdom that `SELECT *` is bad practice, if only because it gives the caller no idea of what they will receive.